### PR TITLE
level 3 playable

### PR DESCRIPTION
**Pull request by jeaniewu**
*added lever animation and sprites
*fix audio source for ghostWall
*level 3b button doesnt act out

### CODE REVIEW

File: Assets/Scripts/receiveSignal/receiveSignalGhostWall.cs [hunk 1]
```diff
@@ -3,29 +3,36 @@
 
 public class receiveSignalGhostWall : receiveSignal {
 
-    public GameObject ghostWallAudio;
+	public AudioSource[] ghostWallAudioSources;
     private ParticleSystem particleSystem;
 	private ParticleSystemRenderer renderer;
```
Comment (maiahta) [inline Assets/Scripts/receiveSignal/receiveSignalGhostWall.cs:8]: could someone fix the indenting :D :D

**Comment by maiahta** [comment]
I APPROVE :+1:

File: Assets/Scripts/receiveSignal/receiveSignalLockedDoor.cs [hunk 2]
```diff
@@ -6,22 +6,27 @@ public class receiveSignalLockedDoor : receiveSignal {
...
     void Start() {
         animator = GetComponent<Animator>();
+		collider = GetComponent <BoxCollider2D> ();
     }
 
 
     public override void activate() {
         animator.SetBool("open", true);
+		collider.enabled = false;
         if (doorToNextLevel != null) {
```
Comment (jayfriso) [inline Assets/Scripts/receiveSignal/receiveSignalLockedDoor.cs:22]: You should see if you can get rid of the doorToNextLevel boolean. We can maybe just haved the collider get in the way so you can't interact with the door.

**Comment by jayfriso** [comment]
Looks good beside the doorToNextLevel thing (which is code I wrote lol).

Merge and then see if you can refactor that.

**Pull request merged**